packages/examples/packages/transaction-insights/src/index.ts

```diff
@@ -30,10 +30,17 @@ export const onTransaction: OnTransactionHandler = async ({ transaction }) => {
   ) {
     const type = decodeData(transaction.data);
     return {
-      content: panel([text('**Transaction type:**'), text(type)]),
+      content: panel([
+        row('From Address', address(transaction.from)),
+        row(
+          'To Address',
+          transaction.to ? address(transaction.to) : text('None'),
+        ),
```

Comment:
Isn't it implied by the `address` component that these are addresses?

```suggestion
        row('From', address(transaction.from)),
        row(
          'To',
          transaction.to ? address(transaction.to) : text('None'),
        ),
```

Reply:
yeah!